### PR TITLE
Add login auth

### PR DESCRIPTION
**Pull request by ty10r**
Added very basic OAuth github login/authentication.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,24 +1,61 @@
...
+GH_CLIENT_ID = '5e9a600fa31f8cc796d3'
+GH_CLIENT_SECRET = '35ca42662c9493ef03934626738da50f837b8117'
+
+
 app = Flask(__name__)
 
 # Move to env
 app.debug = True
-
+app.secret_key = '\xfa\x15\x8a\xc9\t\x9b\xb7\xfa\xc5\x10\xfbD\x8f\x88Y\x96\xfe\xabet\x9f\x10(\xb5'
```
Comment (mick) [inline app.py]: we should move the app secret out of the repo and into a ENV var

**Comment by migurski** [comment]
To fix the app secret, use `git --amend` and then `git push --force` which will allow you to overwrite the commit here, and flush the secret from the git repository.

**Comment by migurski** [comment]
Also: I tested the login, and it put me through all the right moves on Github. Great! Is it possible to log out?

File: app.py (hunk 1 above)
Comment (ty10r) [inline app.py]: Will do. This secret is also temporary until the app is hosted.

**Comment by ty10r** [comment]
@migurski Thanks! Log out is to come. I've ran into a couple snags while working through this so it's still very basic.
I sent the pull request to get it started like you suggested.
I want to add team verification, log in page, and log out. In that order.

**Comment by mick** [comment]
Great! This worked well for me locally, and looks good.

As the amount of logic in app.py is grow lets look for some opportunities to refactor that. Flask apps have a tendency to gather into 1 file a bit too much, imho.

**Pull request merged**